### PR TITLE
koord-scheduler: move set parent into addChildGroupQuotaInfo

### PR DESCRIPTION
**Pull request by kingeasternsun**
Signed-off-by: kingeasternsun <kingeasternsun@gmail.com>

### Ⅰ. Describe what this PR does

When adding new child node to parent node in tree data structure, It would be better to put the code that set child's parent and the code set parent's child together.

### Ⅱ. Does this pull request fix one issue?

none

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

**Comment by kingeasternsun** [comment]
Thanks for your reviews @eahydra

**Comment by eahydra** [comment]
> Thanks for your reviews @eahydra

Thanks for your contribution @kingeasternsun.
If you are interested in this work, you can take a look at the related implementation as a whole, and you can make some optimizations for the implementation.

File: pkg/scheduler/plugins/elasticquota/core/quota_info.go [hunk 1]
```diff
@@ -383,6 +383,7 @@ func NewQuotaTopoNode(quotaInfo *QuotaInfo) *QuotaTopoNode {
 }
 
 func (qtn *QuotaTopoNode) addChildGroupQuotaInfo(childNode *QuotaTopoNode) {
+	childNode.parQuotaTopoNode = qtn
```
Comment (eahydra) [inline pkg/scheduler/plugins/elasticquota/core/quota_info.go:386]: From the name of this function, the original implementation is OK. On the whole, this function is not very necessary.

**Comment by kingeasternsun** [comment]
> > Thanks for your reviews @eahydra
> 
> Thanks for your contribution @kingeasternsun. If you are interested in this work, you can take a look at the related implementation as a whole, and you can make some optimizations for the implementation.

Thanks for your suggestion, I will reconsider it.

**Pull request closed without merging**